content/uiux/journey-map/journey-map.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Journey Maps in UX'
+Description: 'The visualization of the process that a person goes through in order to accomplish a goal with the intent of improving the overall user experience by providing a holistic view of the user's journey. '
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'UI'
+- 'UX'
+- 'Design'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A journey map is a visualization of the process that a person goes through in order to accomplish a goal. In basic form the journey mapping starts by compiling a series of user actions into a timeline. Then, the timeline is fleshed out with user thoughts and emotions in order to create a narrative which best fits that users persona throughout the customers journey. 
+
+## How Journey map is used in UX**
+ The Journey Map, helps teams understand and visualize the end-to-end experience of a user as they interact with a product or service. They provide a comprehensive view of the user's interactions, emotions, and touchpoints throughout their journey. These maps are a versatile tool in UX design that aids in understanding, visualizing, and improving the overall user experience by providing a holistic view of the user's journey.
+![Journey map](../../../media/journey_map#2.jpg)
```

Comment:
The link should look like this instead of path. Also instead of that `#` use `-` in the image name.

Reply:
done